js/jquery.ntm.js

```diff
@@ -53,7 +53,13 @@
                     }
                 }
 
-                //Thinesh - Explicitly hide the code that children nodes are expanded automatically next refresh.
+                //Thinesh - Add id for parent node
+                if (options.autoParentDetection) {
+                    parent.attr('id','parentnode-' + parentindex);

```

Comment:
where is parentindex 1. declared and 2. initialized? 
Also parentindex needs to be declared as parentIndex.